View/View.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import Form from '../Form/Form'
+import FlashCards from '../FlashCard/FlashCards'
+
+const View = (props) => {
+  const cardView = <FlashCards data={props.data.state.data} />
```

Comment:
Instead of making a whole new `View` component, just have the `App` component deal with it for now.